src/Command/MeilisearchCreateCommand.php

```diff
@@ -9,6 +9,7 @@
 use Meilisearch\Bundle\Exception\TaskException;
 use Meilisearch\Bundle\Model\Aggregator;
 use Meilisearch\Bundle\SearchService;
+use Meilisearch\Bundle\SettingsProvider;
```

Comment:
Is there any requirement to have the `Interface` suffix? 

`SettingsProviderInterface`

Reply:
SearchService is also an interface, so avoided

Reply:
Like this: https://www.php-fig.org/bylaws/psr-naming-conventions/

Reply:
I think we could have the Interface suffix on both. But it could be done later I'll create a new issue to make this repo PSR complaint https://www.php-fig.org/psr/